pylot/planning/messages.py

```diff
@@ -2,21 +2,28 @@
 
 
 class WaypointsMessage(erdos.Message):
-    """Message class to be used to send waypoints.
+    """Message class to be used to send waypoints. Optionally can also send
+    a target speed for each waypoint.
```

Comment:
Can you please follow the Google comment guidelines https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html ? (i.e., The comment should start with a single line description of the class. You can move the second sentence to a separate line)